Makefile

```diff
@@ -11,9 +11,9 @@ venv:
 .PHONY: dockerengine
 dockerengine:
```

Comment:
OK, at this point, I think we should 
- drop this rule, it's no longer relevant. You will also have to remove it from the docs (installation)
- Remove travis provided docker in the travis yaml file. 
- Have make deps do the docker installation (as it is doing now)